src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -811,6 +811,37 @@ Log.Async.Enable = 0
 
 Metric.Enable = 0
 
+#
+#    Metric.InfluxDB
+#        Description: Connection settings for InfluxDB.
+#
+#        For InfluxDB v1:
+#                     Only fill in Metric.InfluxDB.Connection.
+#
+#        Example:
+#                     Metric.InfluxDB.Connection = "hostname;port;database"
+#
+#        For InfluxDB v2:
+#                     Fill in every field.
+#
+#        NOTE:        Currently, Grafana will not work with the provided json files to visualize
+#                     data from InfluxDB v2.
+#
+#        Example:
+#                     Metric.InfluxDB.Connection = "hostname;port"
+#                     Metric.InfluxDB.v2 = 0 - (Disabled)
+#                                          1 - (Enabled)
+#                     Metric.InfluxDB.Org = "my-org"
+#                     Metric.InfluxDB.Bucket = "my-bucket"
+#                     Metric.InfluxDB.Token = "my-token"
+#
+
+Metric.InfluxDB.Connection = "127.0.0.1;8086;worldserver"
+Metric.InfluxDB.v2 = 0
```

Comment:
Should this be defaulted to enabled if v1 is meant to be legacy support?

Reply:
Maybe a note should be made to swap it once the issue is resolved?

Reply:
there is a disclaimer in the config info, that's enough I recon. But can be added in the grafana issue for sure